.configure

```diff
@@ -32,9 +32,12 @@
       "file": "iOS/JPiOS/Jetpack-Secrets.swift",
       "destination": "~/.configure/wordpress-ios/secrets/Jetpack-Secrets.swift",
       "encrypt": true
+    },
+    {
+      "file": "iOS/Reader/Reader-Secrets.swift",
```

Comment:
Something not obvious to me was that the source file cannot be called `Secrets.swfti`, even though the Reader prefix seems redundant in a folder called Reader.

The reason for that is that when `configure` encrypts the files locally, it uses the file name. And given we already have a source file called `Secrets.swift` we cannot name this new one that.